src/rt-app_types.h

```diff
@@ -176,6 +176,8 @@ typedef struct _sched_data_t {
 	unsigned long runtime;
 	unsigned long deadline;
 	unsigned long period;
+	int boost;
+	int cap;
```

Comment:
ditto

Reply:
The idea was to expose a more generic concept which, if we have uclamp, translates in `clamp_{min,max}`.
But I get your point and I agree that at this stage is generates more confusion for nothing.

Reply:
Done